src/main/kotlin/no/nav/pia/sykefravarsstatistikk/persistering/KvartalsvisSykefraværshistorikkService.kt

```diff
@@ -128,16 +130,31 @@ class KvartalsvisSykefraværshistorikkService(
         }
         response.add(umaskertLandstatistikk.tilMaskertDto(type = LAND.name, label = "Norge"))
 
-        val umaskertSektorstatistikk = hentSykefraværsstatistikkSektor(
-            sektor = overordnetEnhet.sektor,
-            førsteÅrstalOgKvartal = førsteKvartal,
-        ).ifEmpty {
-            return Feil(
-                feilmelding = "Ingen sektorstatistikk funnet",
-                httpStatusCode = HttpStatusCode.BadRequest,
-            ).left()
+        if (overordnetEnhet.sektor !== null) {
+            val umaskertSektorstatistikk = hentSykefraværsstatistikkSektor(
+                sektor = overordnetEnhet.sektor,
+                førsteÅrstalOgKvartal = førsteKvartal,
+            ).ifEmpty {
+                return Feil(
```

Comment:
Er dette fortsatt en feil?

Reply:
Tja, dette skal ikke skje egentlig. Feilen oppstår hvis vi ikke har statistikk for virksomhetens sektor (og ikke hvis vi ikke har sektor til virksomheten) 
Om dette skulle dukke opp i loggene betyr det at vi ikke har importert alle sektorer (det er bare fire av dem), noe som må fikses etter feilen oppstår

Reply:
Legger til en Warn logg i det tilfelle, "Ingen sektorstatistikk funnet for sektor X"